artview/components/display.py

```diff
@@ -0,0 +1,297 @@
+"""
+display.py
```

Comment:
I know this is kind of unimportant, but I really think the name **display.py** does not reflect the file content.